pkg/ui/workspaces/cluster-ui/src/store/databaseDetails/databaseDetails.selectors.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import { localStorageSelector } from "../utils/selectors";
+import { LocalStorageKeys } from "../localStorage";
+import { AppState } from "../reducers";
+
+export const selectDatabaseDetails = (state: AppState) => {
+  return state.adminUI.databaseDetails;
```

Comment:
nit: IDK if this needs to be a function - wdyt?

Reply:
Agree, just using the field now.